solutions/landing-zone/README.md

```diff
@@ -88,13 +88,19 @@ To deploy this Landing Zone you will first need to create a Bootstrap project wi
 
 0. Set Default Logging Storage Location.
 
-    This command will ensure that the default logging buckets that are generated with a new project are set to the selected region instead of the default location `global`.
+    This command will ensure that the default logging buckets that are generated with a new project (organization wide) are set to the selected region instead of the default location `global`.
 
     ```
+    export REGION=northamerica-northeast1
+    export PROJECT_ID=$(gcloud config list --format 'value(core.project)')
+    export ORG_ID=$(gcloud projects get-ancestors $PROJECT_ID --format='get(id)' | tail -1)
+    export EMAIL=your-super-admin@email.com
```

Comment:
This line is meant to be modified - the 4 exports were taken from a running system - the block as a whole is not pasted into the shell

Reply:
I inadvertently kicked in global code blocks because of the 4 char spacing when adding headers - removed headers

and fixed the existing cloud alpha logging to gcloud